### ISSUE TITLE
Request to consider a setting for recursive flow variable rendering for enhanced productivity

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#3450]
### Feature description

"As almost all our flow variables must be rendered, they always need to call the render function to use them, which is verbose."

**Comment by acmaste** [comment]
Here is a small example flow using nested flow variable references without relying on any fully recursive renders:

```yaml
id: flow-variable-example
namespace: company.team

inputs:
  - id: date
    type: DATETIME
    required: false

  - id: user
    type: STRING
    defaults: Rick

variables:
  date_raw: "{{ inputs.date ?? execution.startDate }}"
  day_of_week: "{{ render(vars.date_raw, false) | date('EEEE') }}"
  full_date: "{{ render(vars.day_of_week, false) }}, {{ render(vars.date_raw, false) | date('yyyy-MM-dd') }}"
  greeting: "Hello, {{ inputs.user }} on {{ render(vars.full_date, false) }}"

tasks:
  - id: hello
    type: io.kestra.core.tasks.log.Log
    message: "{{render(vars.greeting, false)}}"
```

**Comment by anna-geller** [comment]
@acmaste we can consider adding a `renderOnce()` function, which would be equivalent to `render(vars.your_var, false)` but less verbose. Would that help?

**Comment by brian-mulier-p** [comment]
Hello ! FYI the renderOnce function is available starting from 0.16.0 :+1:
